stetho/src/main/java/com/facebook/stetho/inspector/network/BasePrettyPrinterFactory.java

```diff
@@ -0,0 +1,35 @@
+package com.facebook.stetho.inspector.network;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintWriter;
+
+import com.facebook.stetho.common.Util;
+import com.facebook.stetho.inspector.network.AsyncPrettyPrinter;
+import com.facebook.stetho.inspector.network.AsyncPrettyPrinterFactory;
+
+/**
+ * Implementation of {@link AsyncPrettyPrinterFactory} which generates asynchronous pretty printers
```

Comment:
This documentation and the name is incorrect.  It should be `AbstractSchemaDownloadingPrettyPrinterFactory` or some such thing (maybe a shorter name would be wise *grin*)